R/scale_ggsurvfit.R

```diff
@@ -34,8 +34,20 @@
 #' # specify additional scales
 #' ggsurvfit +
 #'   scale_ggsurvfit(x_scales = list(breaks = seq(0, 8, by = 2)))
+#'
+#' # Specical case: in the risk table, large numbers (with more than 4 digits) may not
```

Comment:
Instead of putting this text as a comment in the examples section, let's add it to a new section the help file using the `#' @details` tag.